static/app/views/issueDetails/streamline/sidebar/groupActivityItem.tsx

```diff
@@ -284,30 +284,54 @@ export default function getGroupActivityItem(
           }),
         };
       case GroupActivityType.SET_RESOLVED_IN_RELEASE: {
-        // Resolved in the next release
+        const hasIntegration =
+          'integration_id' in activity.data && activity.data.integration_id;
+        const integrationLink = hasIntegration ? (
+          <Link
+            to={`/settings/${organization.slug}/integrations/${activity.data.provider_key}/${activity.data.integration_id}/`}
+          >
+            {activity.data.provider}
+          </Link>
+        ) : null;
+        const viaIntegration = hasIntegration
+          ? tct(' via [integration]', {integration: integrationLink})
+          : '';
+
         if ('current_release_version' in activity.data) {
           const currentVersion = activity.data.current_release_version;
           return {
             title: t('Resolved'),
-            message: tct('by [author] in releases greater than [version] [semver]', {
+            message: tct(
+              'by [author] in releases greater than [version] [semver][viaIntegration]',
```

Comment:
Would it be better to do `by [author] in releases greater than [version] [semver] via [ingregration]` instead? That should translate better. Requires you to duplicate the message, but I think that's technically the correct way to do it